src/client/commands/submit.rs

```diff
@@ -45,6 +46,9 @@ pub struct SubmitOpts {
     #[clap(long)]
     name: Option<String>,
 
+    #[clap(long)]
+    cwd: Option<String>,
```

Comment:
It should be not String but PathBuf directly (I am sorry, I gave you a bad example in stdout/stderr, that should be changed too)

Reply:
I'm gonna change it all :D